src/main/java/org/folio/consortia/service/impl/TenantServiceImpl.java

```diff
@@ -42,6 +52,11 @@ public class TenantServiceImpl implements TenantService {
   private final FolioExecutionContext folioExecutionContext;
   private final FolioModuleMetadata folioMetadata;
   private final ConsortiaConfigurationClient configurationClient;
+  private final UsersClient usersClient;
```

Comment:
There is some abstraction layering issue here, TenantService can inject TenantDao or TenantClient, but for interaction with users we should inject UserService or probably UserTenantService. I propose to introduce new UserService that will be injected in both UserTenantService and TenantService